prisma/seed/data/roles.ts

```diff
@@ -8,4 +8,8 @@ export default [
         description:
             "Advanced Projects - Apps having both Front-end and Back-end components (FULL STACK)",
```

Comment:
If you have time could you also please update the admin team description? Just something like "Chingu admins"